lib/jxl/enc_frame.cc

```diff
@@ -1161,15 +1165,15 @@ Status EncodeFrame(const CompressParams& cparams_orig,
       CopyImageTo(ib.color(), &opsin);
     }
     if (ib.HasAlpha() && !ib.AlphaIsPremultiplied() &&
-        (frame_header->encoding == FrameEncoding::kVarDCT ||
-         cparams.quality_pair.first < 100) &&
-        !cparams.keep_invisible &&
+        cparams.keep_invisible == Override::kOff &&
```

Comment:
there's an ApplyOverride method that might be better

Reply:
Right, that way the default logic can be put in enc_frame instead of in cjxl. Updated.